integration-tests/utils/templates/secrets.go

```diff
@@ -24,7 +24,8 @@ Keystore = '................' # Required
 
 [Mercury.Credentials.cred1]
 # URL = 'http://host.docker.internal:3000/reports'
-URL = 'localhost:1338'
+LegacyURL = 'http://localhost:53299'
+URL = 'http://localhost:53299'
```

Comment:
@skudasov @anirudhwarrier based on my local testing, the mock server uses this port, and I also added LegacyURL to support mercury v0.2. Let me know if these changes are legit. If not, I will revert. Thanks!